docs/Installation.md

```diff
@@ -703,6 +704,36 @@ sudo -u postgres psql -f ./cleanup-postgres.sql # MUST BE VERIFIED!
 
 Remove the database file and recreate it following the installation instructions above.
 
+## D. Optional features
+
+### D.1 Metrics
+
+Statsd metrics are available, to enable the feature install the additional
+`Net::Statsd` module. See the [configuration][Backend configuration] to
+configure the receiver.
+
+The list of metrics is available in the [Telemetry document][metrics];
+
+### D.1.1 Installation on Centos
+
+```sh
+sudo cpanm Net::Statsd
+```
+
+### D.1.2 Installation on Debian / Ubuntu
+
+
+```sh
+sudo apt install libnet-statsd-perl
+```
+
+### D.1.3 Installation on Freebsd
+
+```sh
+# For metrics feature

```

Comment:
A detail, but this comment could preferably be removed.

Reply:
done